Documentation/guides/vscode-support.md

```diff
@@ -0,0 +1,86 @@
+# VSCode Support
+
+Xamarin.Android itself can be developed within VSCode. There is
+a workspace included in the repo `Xamarin.Android.code-workspace`.
+The required extensions should be installed when you open the
+workspace.
+
+## Building Xamarin.Android
+
+Follow the instructions [here]() to build Xamarin.Android on the
+command line first. Then open the `Xamarin.Android.code-workspace`
+in vscode.
+
+## Running/Debugging Unit Tests
+
+Xamarin.Android uses the `nxunit-explorer` extension to allow you to run and
+debug unit tests. This extension should be installed automatically.
+In order to run or debug the tests you will need to set a path
+to the `nunit-console.exe` file. This is will be located in your
+global nuget cache, usually  in `$HOME/.nuget`.
+
+The setting you need to set is `nxunitExplorer.nunit`. This will need
+to be done in your `Preferences->Settings` in VSCode. This will save
+the setting globally so it will be available when ever you open
+Xamarin.Android.
+
+Alternatively you can add something like this to the `Xamarin.Android.code-workspace`.
+
+```json
+"nxunitExplorer.nunit": "/Users/<user>/.nuget/packages/nunit.consolerunner/3.11.1/tools/nunit3-console.exe"
```

Comment:
Can [environment variables](https://code.visualstudio.com/docs/editor/variables-reference#_environment-variables) be used?

```json
"nxunitExplorer.nunit": "${env:HOME}/.nuget/packages/nunit.consolerunner/3.11.1/tools/nunit3-console.exe"
```

Assuming that works at all, the problem with it will be that it's specific to Linux & macOS, not Windows.  Though maybe *both* could be used, in a "fallback" affair?

```json
"nxunitExplorer.nunit": "${env:USERPROFILE}${env:HOME}/.nuget/packages/nunit.consolerunner/3.11.1/tools/nunit3-console.exe"
```